protected/controllers/AdminController.php

```diff
@@ -1602,16 +1601,23 @@ public function actionEditCommissioningBodyService()
                     $transaction->rollback();
                     throw $e;
                 }
-
-
                 $this->redirect($return_url);
             }
         }
-
+        $criteria = new CDbCriteria();
+        $criteria->order = 't.name asc';
+        if ($commissioning_bt) {
+            $criteria->addColumnCondition(array('commissioning_body_type_id' => $commissioning_bt->id));
+        }
+        $data = CHtml::listData(CommissioningBody::model()->findAll($criteria), 'id', 'name');
+        if (empty($data)) {
+          array_push($errors, array('No Local Authorities type commissioning body is defined, please contact admin user to fix this problem.'));
```

Comment:
Hang on, doesn't this error appear on an Admin page? Why should they contact an admin?

Reply:
I am trying to fix the empty list of commissioning body in LA Admin page, now I realise that it use the same function in admin page.

Reply:
Seems we can only add the commissioning bodies in admin>core. So I thought they need contact admin.